messages/core_write_access.md

```diff
@@ -0,0 +1,14 @@
+Hello {NAME},
+
+{some information about why they are getting access now}
+
+You can see the formal description of the subpackage maintainers role at http://www.astropy.org/team.html.  But there are some other guidelines to keep in mind, which I'm describing here.  In general you should only merge PRs that are in the domain of your subpackage(s).  This is a guideline rather than a firm rule: if you see something that's straightforward and obvious (e.g., a minor typo fix that's already been approved and the tests have passed but no one hit the merge button), you should feel free to help with that sort of thing even in other packages. Similarly, in areas where there's conflict/uncertainty, it's best to defer to a more senior maintainer (if there is one) or if absolutely necessary the Coordination Committee... but in cases where there's no obvious reason to question it (particularly if the lead is the PR author), it is also fine to merge things when it seems reasonable.
```

Comment:
Instead of saying about the 'more senior maintainer' bit I wonder if it's better to say that it's important in difficult cases to get approval from *multiple* maintainers (this ties in to the policy @taldcroft has been writing). Actually maybe thinking about this this email should basically link to @taldcroft's policy document? (in other words, all guidelines should be in policies and you can link to them rather than repeat them?). Basically this email should probably be mostly links :)

Reply:
While in a general sense I agree, I think it's important *in this email* to emphasize the value of judgement and community-building, since that's what the maintainers are there to do. So I totally agree with the "more senior maintainer" edit, but am less convinced we should lean on the policy doc links, since that takes agency away from the maintainers.  In any event, the policy docs don't exist yet, so my thinking is that I'll do the first half of this and we can defer the rest to when that #100 is in!